src/plays/why-typescript/Readme.md

```diff
@@ -0,0 +1 @@
+# Why Typescript
```

Comment:
Put some more details in the Readme. You can refer other play readmes

Reply:
done